robyn/scaffold/structured/sqlalchemy/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
oof. this is not enough unfortunately 😅

Reply:
This is generated by `alemic init`, not necessary. Removed this README